src/main/java/com/google/devtools/build/lib/remote/util/Utils.java

```diff
@@ -707,4 +708,25 @@ public static void waitForBulkTransfer(
       throw bulkTransferException;
     }
   }
+
+  /**
+   * Converts from a Starlark string, which contains raw bytes, to a Protobuf
+   * string, which contains Unicode code points.
+   */
+  public static String starlarkStringToProto(String starlarkString) {
```

Comment:
Renamed to `starlarkToUnicode()` and `unicodeToStarlark()`.

I'm somewhat hopeful that a future iteration of Bazel might change Starlark strings from `[]byte` to `[]char`, so trying to avoid putting specific encodings into the function names.

Reply:
OK, I've renamed these to `decodeBytestringUtf8()` and `encodeBytestringUtf8()`. I also added some examples in the docstrings.

Reply:
Thanks, this looks good now.